Packs/Iboss/Integrations/Iboss/Iboss.py

```diff
@@ -0,0 +1,1329 @@
+            url=entry, current_policy_being_edited=current_policy_being_edited,
+            allow_keyword=allow_keyword,
+            direction=direction, end_port=end_port, apply_global=apply_global, is_regex=is_regex,
+            is_timed_url=is_timed_url, note=note,
+            priority=priority, start_port=start_port, time_url_expires_in_minutes=time_url_expires_in_minutes)
+        command_results.append(CommandResults(
+            readable_output=result['message'],
+            outputs_prefix='iboss.AddEntityToBlockList',
+            outputs_key_field='message',
+            outputs=result,
+        ))
+
+    return command_results
+
+
+@metadata_collector.command(
+    command_name='iboss-remove-entity-from-block-list',
+    description="Removes domains, IPs, and/or URLs to a block list.",
+    outputs_prefix='iboss.RemoveEntityFromBlockList',
+    inputs_list=[
+        InputArgument(name="entity", description="Domains, IPs, and/or URLs to remove from a block list.", required=True,
+                      is_array=True),
+        InputArgument(name="current_policy_being_edited", description="The group/policy number to update.", default="1")
+    ],
+    outputs_list=[
+        OutputArgument(name="message", description="Operation result.", output_type=str)
+    ])
+def remove_entity_from_block_list_command(client: Client, args: Dict[str, Any]) -> list[CommandResults]:
+    """Removes entities from a block list
+
+     Args:
+        client (Client): iboss client to use.
+        entity: required. Domains, IPs, and/or URLs to remove from a block list.
+        current_policy_being_edited: required. The group/policy number to update. defaultValue=1.
+
+    Returns:
+        A ``CommandResults`` object that is then passed to ``return_results``, that contains the status of the removed entity.
+
+    Context Outputs:
+        message (str): Update result.
+    """
+
+    entities = _get_validate_argument(
+        "entity", args, validator=lambda x: x and len(x) > 0, message="value is not specified")
+
+    current_policy_being_edited = _get_validate_argument(
+        "current_policy_being_edited", args, validator=lambda x: (x or x == 0) and x > 0, message="value must be >= 0",
+        return_type=int)
+
+    command_results = []
+    for entry in argToList(entities):
+        result = client.remove_entity_from_block_list(url=entry,
+                                                      current_policy_being_edited=current_policy_being_edited)
+
+        # Update with friendlier message when user attempts to remove entry that is not present on list
+        if result.get("errorCode", -1) == 0 and result.get("message", "") == "Failed to remove URL.":
+            result["message"] = "URL not found in list."
+
+        command_results.append(CommandResults(
+            readable_output=result["message"],
+            outputs_prefix="iboss.RemoveEntityFromBlockList",
+            outputs_key_field="message",
+            outputs=result,
```

Comment:
So in this case, the result really only returns a message. I did modify the message a bit to be more friendly. Do you think it would be useful to include parameters that the user sent to the API in the return message?

Reply:
As we discussed offline - the improved message would be sufficient. 
Anyway, this is something you could change\add in the future if it would be needed.